pages/merchant/Merchant.qml

```diff
@@ -0,0 +1,697 @@
+                                return qsTr("Currently monitoring incoming transactions, none found yet.");
+                            } else {
+                                return ""
+                            }
+                        }
+                        onHideAmountToggled: {
+                            if(root.hiddenAmounts.indexOf(txid) < 0){
+                                root.hiddenAmounts.push(txid);
+                            } else {
+                                root.hiddenAmounts = root.hiddenAmounts.filter(function(_txid) { return _txid !== txid });
+                            }
+                        }
+                    }
+                }
+            }
+
+            DropShadow {
+                anchors.fill: source
+                cached: true
+                horizontalOffset: 3
+                verticalOffset: 3
+                radius: 8.0
+                samples: 16
+                color: "#20000000"
+                smooth: true
+                source: tracker
+            }
+
+            Rectangle {
+                id: qrImg
+                color: "white"
+
+                anchors.right: parent.right
+                anchors.top: parent.top
+
+                height: root.qrCodeSize
+                width: root.qrCodeSize
+
+                Layout.maximumWidth: root.qrCodeSize
+                Layout.preferredHeight: width
+                radius: 5
+
+                Image {
+                    id: qrCode
+                    anchors.fill: parent
+                    anchors.margins: 1 * scaleRatio
+
+                    smooth: false
+                    fillMode: Image.PreserveAspectFit
+                    source: "image://qrcode/" + TxUtils.makeQRCodeString(appWindow.current_address, amountToReceive.text)
+
+                    MouseArea {
+                        anchors.fill: parent
+                        acceptedButtons: Qt.RightButton
+                        onClicked: {
+                            if (mouse.button == Qt.RightButton){
+                                qrMenu.x = this.mouseX;
+                                qrMenu.y = this.mouseY;
+                                qrMenu.open()
+                            }
+                        }
+                        onPressAndHold: qrFileDialog.open()
+                    }
+                }
+
+                Menu {
+                    id: qrMenu
+                    title: "QrCode"
+
+                    MenuItem {
+                        text: qsTr("Save As") + translationManager.emptyString;
+                        onTriggered: qrFileDialog.open()
+                    }
+                }
+            }
+
+            DropShadow {
+                anchors.fill: source
+                cached: true
+                horizontalOffset: 3
+                verticalOffset: 3
+                radius: 8.0
+                samples: 16
+                color: "#30000000"
```

Comment:
Transparent?

Reply:
Slightly transparent. Try `#FF000000` and compare.

Reply:
:ok_hand: